### PR TITLE
Multifile

### PR DESCRIPTION
**Pull request by rsdamm**
fixed README.  Added code to calculated load time and filesize in kb

### CODE REVIEW

File: filewrangle.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    
+  #for filename in filenames:
+  filenames = os.listdir(from_dir)
+  
+  start_time = datetime.now()  
+  for filename in filenames:  
+    from_dir_filename = os.path.join(from_dir, filename)
+    post_dir_filename = os.path.join(post_dir, filename)
+    object_key = get_key(filename)  
+    total_time_to_upload = 0
```
Comment (marinusd) [inline filewrangle.py]: Should total_time_to_upload be set every time inside the loop, or initialized once before the loop?

File: filewrangle.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        
+  if fromdir and postdir and bucketname:
+    print ("Source file directory: %s " % fromdir)   
+    print ("Post processing directory: %s" % postdir)
+    print ("S3 bucketname: %s" % bucketname)
+    print ("  ")
+    print ("  " )
+    process_files(fromdir, postdir, bucketname)
+  else:
+    print ('\n'.join(result))
```
Comment (marinusd) [inline filewrangle.py:145]: 'result' is declared as an array in line 105, but then never filled, and then it's printed (empty?)  when there are no arguments on the command line.  I don't know what function 'result' is serving.

**Review by marinusd — APPROVED** [review]
Left a couple of comments, there's no functional issue.

File: filewrangle.py (hunk 1 above)
Comment (rsdamm) [reply filewrangle.py]: No. It is making a full path/filename per file.

**Pull request merged**